backend/queries/fellows.js

```diff
@@ -51,54 +43,61 @@ const getFellowByEmail = async (fEmail) => {  // function name breaks convention
   return await db.one(getQuery, { fEmail });
 }
 
-const addFellow = async (user, password) => {
-  const registeredUser = await userQueries.addUser(user.email, password, 'fellow');
+const addFellow = async (userObj, password) => {
+  // adds fellow login to users_data table
+  const registeredUser = await userQueries.addUser(userObj.fEmail, password, 'fellow');
 
+  // continues with adding rest of fellow info
   const postQuery = `
     INSERT INTO fellows (
         f_first_name,
         f_last_name,
         f_email,
         cohort_id
     ) VALUES (
-        $/firstName/,
-        $/lastName/,
-        $/email/,
+        $/fFirstName/,
```

Comment:
Can you please put it back the way it was, for consistency on routes/auth.js

Reply:
reverted user object var names